front/app/api/analytics/types.ts

```diff
@@ -24,7 +24,7 @@ export interface QuerySchema {
         };
   };
   groups?: string | string[];
-  aggregations?: AggregationsConfig;
+  aggregations?: any; // AggregationsConfig | AggregationsConfig[];
```

Comment:
@amanda-anderson I ran out of time to fix these typings. Wonder if you might be able to pick that up?